### PR TITLE
make numbers after candidates customizable

### PR DESCRIPTION
**Pull request by yqrashawn**

### CODE REVIEW

**Comment by dgutov** [comment]
I'm curious, how are you going to make them look?

**Comment by yqrashawn** [comment]
Make it `asdf`. So that I can use select candidates with home row more naturaly.

http://yqrashawn.com/2019/01/15/use-emacs-with-home-row/

**Comment by dgutov** [comment]
All right. Could you add a NEWS entry?

File: NEWS.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Next
 
+* Add custom variable `company-show-numbers-function` to make numbers of
+  candidates customizable.
+
```
Comment (dgutov) [inline NEWS.md]: Extra newline

Comment (yqrashawn) [reply NEWS.md]: Fixed. Sorry for that.

**Comment by dgutov** [comment]
Thank you.

**Pull request merged**